tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/record/ClosedRecordEquivalencyRulesTest.java

```diff
@@ -50,8 +50,6 @@ public void testClosedRecordNegatives() {
         int i = 0;
         validateError(negative, i++, "incompatible types: expected 'AnotherPerson', found 'Person1'", 28, 24);
         validateError(negative, i++, "incompatible types: expected 'AnotherPerson', found 'Person2'", 38, 24);
-        validateError(negative, i++, "incompatible types: expected 'AnotherPerson3', found 'Person1'", 49, 25);
```

Comment:
Please add both negative and positive (compile time and runtime) tests for the record assignability changes introduced in this PR.